less/index.less

```diff
@@ -50,10 +131,278 @@ table {
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+	box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
+html,
+body {
+	height: 100%;
+}
+
+@defaultfontsize: 62.5%;
+@mobilefontsize: 100%;
+
+@mobilebreakpoint: 500px;
+@tabletbreakpoint: 750px;
+@desktopbreakpoint: 1000px;
+
```

Comment:
Good use of variables, I would probably separate the words with a "-" just to make it more readable. So  default-font-size rather than defaultfontsize